graphene_django/converter.py

```diff
@@ -61,6 +60,24 @@ def wrapped_resolver(*args, **kwargs):
         return blank_field_wrapper(resolver)
 
 
+class EnumValueField(Field):
```

Comment:
Looks like this effectively replaces `BlankValueField`. Can you add a comment or something stating as much, and "marking" `BlankValueField` as deprecated? I assume it's effectively only ever been used internally, but we shouldn't remove it without a major version bump, unfortunately.

Reply:
actually, my idea was to inherit from the `BlankValueField` class, but I forgot to. The interesting thing is that I've seen the test that was added when this class was added (`test_choice_enum_blank_value`), but it didn't catch the bug in my code as the test itself was wrong (the field in the test instance was null instead of blank).
I've tested a bug in the test and classes inheritance, so they match my initial idea, but if you want me to move all the logic to the `EnumValueField` and deprecate `BlankValueField`, I can do that